### PR TITLE
Initial CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by sameeradh**
Initial contributing files consisting of basic styleguides and information
Test Case : None
Impact : None

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report for Project. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behavior :computer: :computer:, and find related reports :mag_right:.
+
+Before creating bug reports, please check [this list](#before-submitting-a-bug-report) as you might find out that you don't need to create one.
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
```
Comment (dhruvil009) [inline CONTRIBUTING.md:38]: Should I create a project for this repository?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+#### Before Submitting A Bug Report
+* **Check the list of issues in the dashboard. If the same bug has been described in one of the issues and still open, please put a comment on the same issue. We will get back to you as soon as possible
+
```
Comment (dhruvil009) [inline CONTRIBUTING.md:42]: This can probably come before reporting bugs....?

**Review by dhruvil009 — COMMENTED** [review]
Looks great overall. Nice work 👍

File: CONTRIBUTING.md (hunk 1 above)
Comment (sameeradh) [reply CONTRIBUTING.md:38]: Not needed now, we can discuss later on project and dashboard

File: CONTRIBUTING.md (hunk 2 above)
Comment (sameeradh) [reply CONTRIBUTING.md:42]: yeah

**Pull request merged**